rockcraft/models/project.py

```diff
@@ -133,8 +133,8 @@ class Project(YamlModelMixin, BaseProject):
     description: str  # type: ignore[reportIncompatibleVariableOverride]
     rock_license: str = pydantic.Field(alias="license")
     platforms: dict[str, Any]
-    base: Literal["bare", "ubuntu@20.04", "ubuntu@22.04"]
-    build_base: Literal["ubuntu@20.04", "ubuntu@22.04"] | None
+    base: Literal["bare", "ubuntu@20.04", "ubuntu@22.04", "ubuntu@24.04"]
+    build_base: Literal["ubuntu@20.04", "ubuntu@22.04", "devel"] | None
```

Comment:
The docs should be updated too, including an explanation for this new cool `devel` option :)

Reply:
oh nice
if you like the look of it then I'll work on getting this PR actually ready for merging. I'll ping you when it's done, thanks!